src/layer/poi.js

```diff
@@ -0,0 +1,81 @@
+import * as label from "../constants/label.js";
+import * as Color from "../constants/color.js";
+
+const poiIcon = [
+  "match",
+  ["get", "class"],
+  ["bar", "beer"],
+  "poi_martini_glass",
+  "hospital",
+  "poi_health_cross",
+  "parking",
+  "poi_p",
+  "poi_square_dot", //icon for generic POI, not currently used
+];
+
+const poiFilter = [
+  ">=",
+  ["zoom"],
+  [
+    "match",
+    ["get", "class"],
+    "hospital",
+    15,
+    ["bar", "beer"],
+    16,
+    "parking",
+    17,
+    18,
+  ],
+];
+
+const iconColor = [
+  "match",
+  ["get", "class"],
+  ["bar", "beer"],
```

Comment:
These classes get repeated several times in the same file. It doesn’t seem likely that these constants will ever get reused. Instead, inline these constants, but pull out different constants like `consumerClasses` that can be reused in the filter and color expressions, if not also the icon expression.

Reply:
I've inlined the constants, however I don't see the class constants being re-usuable.  I think zoom decisions are independent of category.